modules/tinymce/src/themes/silver/test/ts/browser/editor/ToxWrappingTest.ts

```diff
@@ -7,7 +7,7 @@ import { Attribute, Class, SugarBody, SugarElement, Traverse } from '@ephox/suga
 import Editor from 'tinymce/core/api/Editor';
 import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('Editor (Silver) test', (success, failure) => {
+UnitTest.asynctest('Editor (Silver) tox wrapping test', (success, failure) => {
```

Comment:
Is there a reason this isn't `browser.tinymce.themes.silver.ToxWrappingTest`? Same question for SilverBespokeButtonsTest and EditorMenubarSettingsTest.

Reply:
We discussed this yesterday, so going to pull up the conversation from slack:
> Question: I've gone on a bit of a :yak: :scissors: tangent here... should I fix all the inconsistent test names in the silver theme?
> I've fixed all the others, just not the silver theme. Remember that 98% of theme are inconsistent

So based on that, we decided we shouldn't worry about the silver names for now and we'll worry about it later:

> …any idea how many there are/how long it would take?
> 
> Looks like there's about 100 of them. I think I'll leave it for another day